poezio/tabs/muctab.py

```diff
@@ -650,32 +650,35 @@ def on_user_join(self, from_nick, affiliation, show, status, role, jid,
     def on_user_nick_change(self, presence, user, from_nick, from_room):
         new_nick = presence.xml.find('{%s}x/{%s}item' %
                                      (NS_MUC_USER, NS_MUC_USER)).attrib['nick']
+        old_color = user.color
+        user.change_nick(new_nick)
```

Comment:
This makes the condition of the next line always fail, since the nick has already changed

Reply:
Ah, so that’s why it happens. Thanks. I may push a fix tomorrow.